mnefun/_mnefun.py

```diff
@@ -822,38 +863,141 @@ def run_sss_positions(fname_in, fname_out, host='kasga', opts='', port=22):
         raise IOError('input file not found: %s' % fname_in)
     if not op.isdir(op.dirname(op.abspath(fname_out))):
         raise IOError('output directory for output file does not exist')
-    fnames_in = [fname_in]
-    for ii in range(1, 11):
-        next_name = op.splitext(fname_in)[0] + '-%s' % ii + '.fif'
-        if op.isfile(next_name):
-            fnames_in.append(next_name)
-        else:
-            break
```

Comment:
I am beginning to wonder if I began working on the same cp of the `run_sss_positions` code!